index.js

```diff
@@ -1,10 +1,11 @@
 const chokidar = require('chokidar');
 const Rsync = require('rsync');
 const debounce = require('debounce');
+const fs = require('fs');
 
 const consoleTimestamp = require('./lib/console-timestamp');
 
-const CONFIG = require('./config.json');
+const CONFIG = require(fs.existsSync('./') ? './config' : './config.json');
```

Comment:
Did you mean exists check for config.js?
And I think it would be better to have only js-config.

Reply:
Yes, that's what I intended to reduce the chances of BC break, but I think you are right. JS only config is a good step ahead.